packages/extension-polkagate/src/hooks/useBalancesOnSingleAssetChain.ts

```
@@ -0,0 +1,144 @@
+        setNewBalances({
+          ED,
+          assetId: NATIVE_TOKEN_ASSET_ID,
+          ...allBalances,
+          chainName,
+          date: Date.now(),
+          decimal,
+          frozenBalance,
+          genesisHash: api.genesisHash.toString(),
+          token
+        });
+        setRefresh && setRefresh(false);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)